Dockerfile.dev

```
@@ -0,0 +1,17 @@
+# Pull image with python 3.9 and nodejs 16
+FROM nikolaik/python-nodejs:python3.9-nodejs16
+# nodemon provides hot reloading support
+RUN npm install -g nodemon
+
+WORKDIR /node-red-contrib-quantum
+
+COPY package*.json .
+
+COPY bin ./bin
+
+RUN npm install && \
+    npm run setup
```

```suggestion
RUN npm run setup
```
This can be simplified to just `npm run setup`, since that command runs install anyway.